core-js/src/main/javascript/lib/i18n/cdf.jquery.i18n.js

```diff
@@ -102,12 +102,12 @@ define([
         var supportedLocale = lines[i];
 
         var lang = supportedLocale.substr(0, supportedLocale.indexOf("="));
-        if (lang === locale && result == null) {
-          result = locale;
+        if (lang.toLowerCase() === locale && result == null) {
```

Comment:
I think it would be best to create a `langLower` variable.

Reply:
Thanks for the feedback

Reply:
I made your suggestions for this.
I am currently analyzing the common-ui file.